client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -44,6 +47,10 @@ module.exports = React.createClass( {
 		};
 	},
 
+	isMobileTouchDevice() {
+		return ! isDesktop() || hasTouch();
```

Comment:
I think this is good for now. We might want to revisit this at some point to make sure we're targeting the right devices (e.g., I think we should be able to support a Surface, but I think we'd be hiding it here). But I don't think we should do that until we get some real user feedback.